apps/src/StudioApp.js

```diff
@@ -2150,6 +2158,22 @@ StudioApp.prototype.configureDom = function (config) {
     trailing: false,
   });
 
+  // Modify throttledRunClick to include metrics logging
+  const originalThrottledRunClick = throttledRunClick;
+  throttledRunClick = () => {
+    originalThrottledRunClick();
+    let eventName;
+    if (!!config.level.isProjectLevel) {
```

Comment:
TODO: I don't think we need boolean coercion here. Looking around, it seems like we can count on isProjectLevel to return a boolean.